tests/Argu.Tests/Tests.fs

```diff
@@ -852,6 +867,11 @@ module ``Argu Tests Main List`` =
         let results = parser.ParseCommandLine (args, raiseOnUsage = false)
         test <@ results.IsUsageRequested @>
 
+    [<Fact>]
+    let ``Required subcommand attribute should fail on missing subcommand and display usage of subcommand and not main command`` () =
```

Comment:
but do you have a missing subcommand in this case? (if that's one of the points, maybe there should be an assert wrt that?)

`Subcommand errors report should take precedence over missing RequiredSubCommand` ?

Reply:
That is right, it's badly formulated.
Does `Should fail if mandatory case is missing on a subcommand and display usage of subcommand and not main command` sound better?